app/services/highlighter/models/ai-highlighter.models.ts

```diff
@@ -28,6 +39,10 @@ export interface IDefaultEventInfo extends IEventInfo {
 
 export enum EGame {
   FORTNITE = 'fortnite',
+  WARZONE = 'warzone',
+  MARVEL_RIVALS = 'marvel_rivals',
+  WAR_THUNDER = 'war_thunder',
+  VALORANT = 'valorant',
```

Comment:
just fyi the enum values can have spaces in them, might be useful for the switch in `highlighter/index.ts`